flake.nix

```diff
@@ -30,15 +30,18 @@
             rustc = rust;
           };
         in
-        {
-          eww = (prev.eww.override { inherit rustPlatform; }).overrideAttrs (old: {
+        rec {
+          # TODO update this to reflect the changes in upstream (nixpkgs), when upstream is updated
+          eww = (prev.eww.override { inherit rustPlatform; withWayland = true; }).overrideAttrs (old: {
             version = self.rev or "dirty";
             src = builtins.path { name = "eww"; path = prev.lib.cleanSource ./.; };
             cargoDeps = rustPlatform.importCargoLock { lockFile = ./Cargo.lock; };
+            cargoBuildNoDefaultFeatures = false;
```

Comment:
Double negative (`cargoBuildNoDefaultFeatures` and `false`) is confusing a bit.

Reply:
Yeah I also don't understand why the authors of `buildRustPackage` have chosen the naming of this...
But AFAIK it's the var that's necessary to use to enable/disable default features

Reply:
It's probably due to how cargo named it (`cargo build --no-default-features`)